devlib/trace/perf.py

```diff
@@ -104,7 +104,11 @@ def start(self):
             self.target.kick_off(command)
 
     def stop(self):
+        self.target.killall('perf', signal='SIGINT',
```

Comment:
The other `self.target.killall('perf'` keeps `SIGKILL` as `perf` is not supposed to be there anyway!